extensions/oidc-client/spi/src/main/java/io/quarkus/oidc/client/spi/TokenProvider.java

```diff
@@ -0,0 +1,13 @@
+package io.quarkus.oidc.client.spi;
+
+import io.smallrye.mutiny.Uni;
+
+public interface TokenProvider {
```

Comment:
Can you add a Javadoc to this interface explaining its purpose?

Reply:
Hi @gastaldi Added a Javadoc, thanks